index.js

```diff
@@ -90,22 +90,23 @@ class GamepadPose {
   }
 }
 class GamepadHapticActuator {
-  constructor(gamepad) {
+  constructor(gamepad, index) {
     this.gamepad = gamepad;
+    this.index = index;
   }
   get type() {
     return 'vibration';
   }
   set type(type) {}
   pulse(value, duration) {
     if (this.gamepad.ontriggerhapticpulse) {
-      this.gamepad.ontriggerhapticpulse(value, duration);
+      this.gamepad.ontriggerhapticpulse(value, duration, this.index);
     }
   }
 }
 class Gamepad {
   constructor(hand, index) {
-    this.id = 'OpenVR Gamepad';
+    this.id = (index < 0) ? 'Controller' : 'OpenVR Gamepad';
```

Comment:
I don’t think this is a good way to distinguish controller type. Could it be an arg coming in?

Reply:
It’s much simpler this way because we can use the same API as OpenVR. See the latest exokit PR